WordPress/build.gradle

```diff
@@ -331,6 +339,22 @@ dependencies {
     implementation (project(path:':libs:editor:WordPressEditor')) {
         exclude group: 'org.wordpress', module: 'utils' 
     }
+
+    // Hermes is a React Native dependency that we somehow lost at runtime
+    // since moving to the Bintray binary for Gutenberg.
+    //
+    // These cannot be defined in the WordPressEditor project itself, or we'll
+    // get a "Direct local .aar file dependencies are not supported when
+    // building an AAR." error.
+    //
+    // FIXME: In the long run, we'll want to read this from the Gutenberg AAR.
+    //
+    // A starting point might be the fact that the
+    // @wordpress_react-native-bridge-release AAR contains only the
+    // -release.aar version of Hermes in its libs folder
```

Comment:
In testing this earlier, I also noticed that the hermes dependency bundled in the aar was way to small of a file (only 22 bytes). 😕